e2e/tests/11_graphql_directive_tests.spec.ts

```diff
@@ -48,22 +49,60 @@ describe("Graphql Directive tests", () => {
         expect(response).to.deep.equal(expectedResponse);
     });
 
-    it("should return hotel by id with orders", async () => {
+    it("should return hotel by id with alias", async () => {
         const hotelId = "hotel_ibis_london_blackfriars";
+        const hotelAlias = "hotelAlias";
+        const expectedResponse = { hotelAlias: hotels.find(h => h.id === hotelId).name };
+
+        const response = await client.request(`{
+            ${hotelAlias}: hotel(id: ${hotelId}) {
+                name
+            }
+        }`);
+        expect(response).to.exist;
+        expect(response).to.deep.equal(expectedResponse);
+    });
+
+    it("should return hotel by id with orders", async () => {
+        const customerId = "1";
+        const reviewsLimit = 2;
+
+        const { firstName, lastName } = customers[customerId];
         const expectedResponse = {
-            name: hotels.find(h => h.id === hotelId).name,
-            orders: orders.filter(o => o.hotelId === hotelId).map(o => ({ startDate: o.startDate, endDate: o.endDate })),
+            customer: {
+                firstName,
+                lastName,
+                orders: orders.filter(o => o.customerId === customerId).map(o => ({
+                    startDate: o.startDate,
+                    endDate: o.endDate,
+                    hotel: {
+                        name: hotels.find(h => h.id === o.hotelId).name,
+                        reviews: reviews.filter(r => r.hotelId === o.hotelId).map(r => ({
+                            author: r.author,
+                            text: r.text,
+                        })),
+                    },
+                })),
+            },
         };
 
         const response = await client.request(`{
```

Comment:
Use `graphql-tag` so you can use variables easily without escaping your own strings and such

Reply:
I'll take care of it in next PR. I want to add support for non-graphql resolvers under graphql directive anyway.